integrations/gitlab/gitlab_integration/ocean.py

```diff
@@ -108,7 +111,9 @@ async def on_start() -> None:
 async def resync_groups(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
     for service in get_cached_all_services():
         async for groups_batch in service.get_all_groups():
-            yield [group.asdict() for group in groups_batch]
+            tasks = [service.enrich_group_with_members(group) for group in groups_batch]
+            enriched_groups = await asyncio.gather(*tasks)
+            yield enriched_groups
```

Comment:
this is not needed any more if i recall correctly, we moved the logic to the member

Reply:
The other way round instead